src/main/java/soot/jimple/internal/JLookupSwitchStmt.java

```diff
@@ -75,6 +76,21 @@ public Object clone() {
     return new JLookupSwitchStmt(getKey(), clonedLookupValues, getTargets(), getDefaultTarget());
   }
 
+  private String toSimpleString() {
+    final char endOfLine = ' ';
+    List<String> cases = lookupValues.stream()
+            .map((lookupValue) -> String.valueOf(lookupValue.value))
+            .distinct()
```

Comment:
please remove the distinct() as lookupvalues should already be distinct. Printing only non-duplicates could make it harder finding a possible bug somewhere that adds duplicate lookupvalues.